content/blog/2021/2021-02-10-explaining-models.md

```diff
@@ -0,0 +1,53 @@
+
+{{% workflow-screenshot src="/blog_img/2021/2021-02-10-data-table.png" %}}
+
+Next, we will build a simple logistic regression predictive model. If inspecting the model in Test and Score, we learn that the model has an AUC of 0.788 and 86 % CA. But what *kind* of a model is this? How does it makes its decisions?
+
+Let us add Explain Model to Logistic Regression and add another connection passing the data. The workflow should look like this:
+
+{{% workflow-screenshot src="/blog_img/2021/2021-02-10-model-workflow.png" %}}
+
+Now open Explain Model. The widget lists top ranked variables, which means they contribute the most to the selected target variable. As we are trying to understand why people leave the company, we have set the target variable to Yes.
+
+The highest ranked variable is OverTime - this is the variable with the highest impact on the prediction. Having a high value of overtime (red dots on the right) means the employee is likely to quit. Also, having low job satisfaction contributes to attrition (blue values on the right). The visualization shows the values that have a high impact on the model on the right and those with a low impact on the left. The color of dot represents the value of the attribute (red for higher values and blue for lower).
+
+{{% workflow-screenshot src="/blog_img/2021/2021-02-10-explain-model.png" %}}
+
+Let us look at, for example, YearsAtCompany. How would we interpret this? The variable has more dots on the left, which means it contributes against the target value (against attrition, i.e. employees will stay). Red dots refer to the value of the attribute. So if the employee is with the company for a long time (high value == red dot), it means it is more likely she will stay (dots are on the left, while our target value is Yes).
+
+Great, now we understand the model and we are ready to make some predictions. Let us load Attrition - Predict with another Dataset widget. We have three new employees, who are described with all the previous variables, but they are lacking Attrition - we do not know, who is more likely to leave.
+
+{{% workflow-screenshot src="/blog_img/2021/2021-02-10-table-predict.png" %}}
+
+Now pass the logistic regression model and the train data set to Explain Predictions. Then select John from the table and pass the selection to Explain Predictions. The widget requires three inputs: the model, training data, and the instance we are predicting (John).
+
+{{% workflow-screenshot src="/blog_img/2021/2021-02-10-predict-workflow.png" %}}
+
+Once again, we are interested in target value Yes. Variables in red increase the probability of the target value (conversely, blue decrease it). The size of the arrow corresponds to the [SHAP value](https://shap.readthedocs.io/en/latest/index.html) - in other words, the larger the arrow the larger the variable's contribution to the target value.
```

Comment:
Maybe just mention that the model predicted John will leave the job (with 77 % probability).